WasteManagement/WasteContainer/doc/spec.md

```diff
@@ -215,7 +221,7 @@ to keep backwards compatibility with current FIWARE reference implementations.
     + Attribute metadata:
         + `dateUpdated`: Timestamp when the last update of the attribute happened.
             + Type: [DateTime](http://schema.org/DateTime)
-        + `TimeInstant` : Timestamp saved by FIWARE's IoT Agents. Note: This attribute has not been harmonized
+        + `TimeInstant` : [Timestamp](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant) saved by FIWARE's IoT Agents. Note: This attribute has not been harmonized
```

Comment:
Since as mentioned offline by @dmoran the needed update will land in `master` in the next release of https://github.com/telefonicaid/iotagent-node-lib, it is OK for me to land this PR and include an issue to point all these references to `master` instead of `develop` not to forget about it as soon as it is possible to do it. In fact I created the following issue for it: https://github.com/Fiware/dataModels/issues/18